### DISCUSSION TITLE
displaying the battery status of the master device on the follower device...

### DISCUSSION THREAD

**Discussion by mukich** [NightscoutFoundation/xDrip#3800] [Ideas]
The need arose when you need to see the battery status of the master device on the follower device через xDrip Cloud. Is it possible to implement in the tab Classic Status Page, according to the type of display of the database? I have a daughter with diabetes who very often forgets to charge her phone, so it would be good if someone implemented such an idea. The periodicity of updating the battery status can be set to 15 minutes.

**Comment by Navid200** [comment]
Thanks for asking this very good question.

I don't believe xDrip can currently upload the phone battery to Nightscout.
It sounds like a reasonable suggestion.

Please let me bring to your attention an alternative.
If you set up to use xDrip sync for following your daughter, I believe it shows the master phone battery.

**Reply by mukich** [reply]
Synchronization is carried out via xDrip Sync.

**Reply by Navid200** [reply]
If you are following your daughter with xDrip sync, you can already see the battery level of her phone on your follower.

**Reply by mukich** [reply]
Do I need to enable something for this?

**Comment by Navid200** [comment]
xDrip already uploads the battery level of the master to Nightscout.
Let me see if I can figure out how to add the ability for a follower to access and show that value.

**Reply by mukich** [reply]
Yes, the battery level of the master device is loaded into Nightscout.

**Comment by Navid200** [comment]
The Nightscout master phone battery level is now shown in the follower xDrip on the status page.  
You need to use the release 2026.07.01 or any after that.
